<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosStats</class>
 <widget class="QWidget" name="PosStats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>1070</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family:'Open Sans, sans-serif';</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="diffplotframe">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4_2">		
          <item>
           <widget class="QCustomPlot" name="diffplot_pos" native="true">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>230</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4_3">
          <item>
           <widget class="QCustomPlot" name="diffplot_weight" native="true">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>230</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
   </item>
	<item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
    <widget class="QFrame" name="frame_6">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QProgressBar" name="strengthBar">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>This is your own estimated network weight compared to the rest of the users staking on the Amsterdex blockchain. </string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">
			QProgressBar {
				border: 2px solid #202b35;				
				background-color: #ffffff;
				text-align: center;
				color: #202b35;
				height: 30px;
				font-family: Ubuntu;
			}
			QProgressBar::chunk {
				background: #83bdb1;
				width: 4px;
			}
		 </string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="format">
         <string>You are not Staking!</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   </layout>
   </item>
   </layout>
  </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qt/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
